etna/records/views/records.py

```diff
@@ -51,17 +57,14 @@ def record_detail_view(request, iaid):
     view is accessible from a fixed URL.
     """
     try:
-        page = Record.search.get(iaid, expand=True)
+        page = Record.api.fetch(iaid=iaid, expand=True)
```

Comment:
Maybe we could use , something like that: 
api_manager = APIManager(Record)
page = api_manager.fetch(iaid=iaid, expand=True)
I find it a bit clearer, for me, it seems like we are using the record to get the APIManager.

Reply:
Moving discussion to https://github.com/nationalarchives/ds-wagtail/pull/234#pullrequestreview-888019163